app/src/main/java/org/oppia/app/walkthrough/WalkthroughFragmentChangeListener.kt

```diff
@@ -3,4 +3,7 @@ package org.oppia.app.walkthrough
 /** Listener for when an activity should change a fragment position. */
 interface WalkthroughFragmentChangeListener {
   fun currentPage(walkthroughPage: Int)
+
+  fun pageWithTopicId(walkthroughPage: Int, topicId: String)
+
```

Comment:
Remove this extra blank line

Reply:
I had mentioned it for line number 7.
To be clear, we never put a blank line before a closing bracket `}`

Reply:
Got it. Is there any doc with guidelines regarding these formatting ?

Reply:
Actually there is no doc because we are managing this from our project itself. But you had mentioned earlier that you are facing issues in reformatting shortcut, for which I remember I informed you to send an email with issue or with video. But I haven't received any email about the issues that we discussed.